### PR TITLE
README: Corrected formatting and typos.

### PR DESCRIPTION
**Pull request by AnonymouX47**
- Proper MarkDown formatting.
- Restructured misleading sentences.﻿
- Corrected typos.
- Camel-cased the library name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,115 +1,108 @@
...
+`Expr` can be imported from `engpy`:
+
+```python
+>>> from engpy import Expr
+```
+
+See the doc file or `Expr`'s documentation.
+
+To interact with `Expr` instances as discrete objects, use the `engpy.interface` module.
+The `interface` module bridges between the `Expr` class and `Expr` datatypes.
```
Comment (AnonymouX47) [inline README.md:33]: @miracle5284 Couldn't figure out what this sentence should mean...

**Comment by AnonymouX47** [comment]
Also, I think the docs should be added to the repo and linked in the README.

File: README.md (hunk 1 above)
Comment (miracle5284) [reply README.md:33]: Just to inform the users of the options of interacting with either Expr directly bypassing strings or  the interface module by import functions

Comment (AnonymouX47) [reply README.md:33]: I guess you meant to say
> Just to inform the users of the options of either interacting with `Expr` directly by passing strings or by using functions provided by the `interface` module.

If that's it, then the sentence on line 33 should definitely be changed... that was what i was trying to point out.
"`Expr` class" and "`Expr` datatype" are _technically_ the same thing.

I suggest it be changed to:
> The `interface` module provides utilities for interacting with `Expr` instances as discrete objects.

Of course, it's been merged now, but the change can be made along with some other later.
Next time, I'll make such PRs to be drafts.

**Pull request merged**